styles/Editor.css

```diff
@@ -939,6 +939,47 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+
+.ProseMirror .table-footer > div:first-child {
+  padding-top: 8px;
+}
+
+.ProseMirror .table-footer > div:not(:first-child)::before {
+  content: '';
+  border-bottom: 1px dashed #C9C9C9;
+  display: block;
+  margin-bottom: 8px;
+}
+
+.ProseMirror .table-footer > div {
```

Comment:
Would that be possible to use a class here too?

Reply:
Done!